freqtrade/resolvers/exchange_resolver.py

```diff
@@ -0,0 +1,56 @@
+"""
+This module loads custom exchanges
+"""
+import logging
+from pathlib import Path
+
+from freqtrade.exchange import Exchange
+from freqtrade.resolvers import IResolver
+
+logger = logging.getLogger(__name__)
+
+
+class ExchangeResolver(IResolver):
+    """
+    This class contains all the logic to load a custom exchange class
+    """
+
+    __slots__ = ['exchange']
+
+    def __init__(self, exchange_name: str, config: dict) -> None:
+        """
+        Load the custom class from config parameter
+        :param config: configuration dictionary or None
+        """
+        self.exchange = self._load_exchange(exchange_name, kwargs={'config': config})
+
+    def _load_exchange(
+            self, exchange_name: str, kwargs: dict) -> Exchange:
+        """
+        Search and loads the specified exchange.
+        :param exchange_name: name of the module to import
+        :param extra_dir: additional directory to search for the given exchange
+        :return: Exchange instance or None
+        """
+        current_path = Path(__file__).parent.parent.joinpath('exchange').resolve()
+
+        abs_paths = [
+            current_path.parent.parent.joinpath('user_data/exchange'),
```

Comment:
you may want to remove this line, since i doubt we'll have user-provided exchange-classes (they should be commited upstream)...
it'll remove the log line 
```
2019-02-18 07:14:02,709 - freqtrade.resolvers.exchange_resolver - WARNING - Path "user_data/exchange" does not exis
```